dynatrace-operator/questions.yml

```diff
@@ -89,12 +89,93 @@ questions:
     type: string
     group: "Global Configuration"
 
+  #################### ClassicFullStack (OPTIONAL) ####################
+
+  - variable: classicFullStack.enabled
+    label: "Enable classic fullstack monitoring"
+    description: "Enables classic fullstack monitoring and rolls out the OneAgent"
+    default: true
+    type: boolean
+    group: "Classic FullStack"
+    show_subquestion_if: true
+    subquestions:
+      - variable: classicFullStack.nodeSelector
+        label: "Node selector to control the selection of nodes"
+        description: "Defines a NodeSelector to customize to which nodes the OneAgent will be rolled out - Please edit as Yaml for the best experience"
+        type: string
+      - variable: classicFullStack.tolerations
+        label: "Custom tolerations for the OneAgent"
+        description: "Defines custom tolerations to the OneAgent - Please edit as Yaml for the best experience - see https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/"
+        default: ""
+        type: string
+      - variable: classicFullStack.waitReadySeconds
+        label: "Wait seconds until ready"
+        description: "Define the time to wait until OneAgent pod is ready after update - defaults to 300s"
+        default: 300
+        type: int
+      - variable: classicFullStack.args
+        label: "Arguments to OneAgent installer"
+        description: "Defines additional arguments which get passed to the OneAgent installer - Please edit as Yaml for the best experience. The expected format is YAML and not a string"
+        default: ""
+        type: string
+      - variable: classicFullStack.env
+        label: "Environment variables for OneAgent"
+        description: "Defines additional environment variables which get passed to the OneAgent - Please edit as Yaml for the best experience"
+        type: string
+      - variable: classicFullStack.priorityClassName
+        label: "Assign priority class to OneAgent pods"
+        description: "Priority class to assign to OneAgent pods, more at https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/"
+        default: ""
+        type: string
+      - variable: classicFullStack.dnsPolicy
+        label: "Set custom DNS Policy"
+        description: "DNS Policy for OneAgent pods. Empty for default (ClusterFirst), more at https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy"
+        default: ""
+        type: string
+      - variable: classicFullStack.labels
+        label: "Custom labels for the OneAgent pods"
+        description: "Defines labels for OneAgent pods to structure workloads as desired - Please edit as Yaml for the best experience"
+        type: string
+      - variable: classicFullStack.useUnprivilegedMode
+        label: "Early Adopter: run unprivileged"
```

Comment:
It's the default now. Is it still early adopter?

Reply:
Oh it isn't - removed early adopter there